src/legacy/core_plugins/data/public/search/aggs/buckets/date_histogram.ts

```diff
@@ -162,10 +180,18 @@ export const dateHistogramBucketAgg = new BucketAggType<IBucketDateHistogramAggC
       default: 'auto',
       options: intervalOptions,
       modifyAggConfigOnSearchRequestStart(agg: IBucketDateHistogramAggConfig) {
-        setBounds(agg, true);
+        const { timefilter } = getQueryService().timefilter;
+        setAggTimeRangeBounds(agg, {
+          force: true,
+          timefilter,
+        });
       },
       write(agg, output, aggs) {
-        setBounds(agg, true);
+        const { timefilter } = getQueryService().timefilter;
```

Comment:
the bounds most likely don't need to be updated on all this steps:
- when getting the buckets
- when field changes
- when we convert aggconfigs to dsl
- when we start the request

Reply:
we can most likely remove it at the time field changes (that was there for the editor)
also i would remove the one at the time we are making request. currently we will convert to dsl and do request at pretty much the same time.